### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by thomashoneyman**
This PR migrates ordered-collections to use GitHub Actions instead of Travis for CI, as tracked for all core libraries in https://github.com/purescript/purescript/issues/3962. It also updates the installation instructions to refer to [Spago](https://github.com/purescript/spago) instead of Bower.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Purely-functional map and set data structures.
 ## Installation
 
 ```
-bower install purescript-ordered-collections
+spago install ordered-collections
 # or
 psc-package install ordered-collections
```
Comment (kl0tl) [inline README.md]: This is the only core library mentioning `psc-package` in its installation instructions, should we remove those lines for consistency?

**Review by kl0tl — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JordanMartinez) [reply README.md]: Let's remove it.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
 [![Latest release](http://img.shields.io/github/release/purescript/purescript-ordered-collections.svg)](https://github.com/purescript/purescript-ordered-collections/releases)
-[![Build status](https://travis-ci.org/purescript/purescript-ordered-collections.svg?branch=master)](https://travis-ci.org/purescript/purescript-ordered-collections)
+[![Build status](https://github.com/purescript/purescript-ordered-collections/workflows/CI/badge.svg?branch=master)](https://github.com/purescript/purescript-ordered-collections/actions?query=workflow%3ACI+branch%3Amaster)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-ordered-collections/badge)](https://pursuit.purescript.org/packages/purescript-ordered-collections)
 
 Purely-functional map and set data structures.
 
-*If you are looking to work with "StrMap"/JavaScript Objects, see <https://github.com/purescript/purescript-foreign-object>*
+_If you are looking to work with "StrMap"/JavaScript Objects, see [`purescript-foreign-object`](https://github.com/purescript/purescript-foreign-object)._
```
Comment (JordanMartinez) [inline README.md]: Should this be updated to no longer refer to `StrMap`?

Comment (thomashoneyman) [reply README.md]: Yea, I think that's a good idea. JavaScript objects captures the point.

**Pull request merged**